sympy/core/evalf.py

```diff
@@ -1390,7 +1390,28 @@ def evalf(self, n=15, subs=None, maxn=100, chop=False, strict=False, quad=None,
 
             verbose=<bool>
                 Print debug information (default=False)
-
+        Note:
+        It is always suggested to use expr.evalf(subs=...) over,
+        expr.subs(...).evalf() or expr.evalf().subs(...) because of the
+        following reasons:-
+        i.  The subs dictionary tells the evalf algorithms which symbols should be
+            replaced with numbers when they are encountered.
+        ii. expr.evalf(subs=...) the expression is run through the evalf algorithm,
+            which takes into account various issues that can lead to loss of
+            significance;
+        iii.expr.evalf(subs=...) avoids the loss of significance caused due to
+            naive substitution.
```

Comment:
These three reasons are just one reason. You could just say "It is suggested to use `expr.evalf(subs=...)` over expr.evalf().subs(...) or expr.subs(...).evalf() because expr.evalf(subs=...) uses the evalf algorithm, which avoids loss of significance issues that can occur with naive substitution".